pkg/util/net/dns/resolveconf.go

```diff
@@ -71,3 +72,23 @@ func ParseSearchDomains(content string) ([]string, error) {
 
 	return searchDomains, nil
 }
+
+func ParseDomain(content string) (string, error) {
+	var domain string
+	domain = ""
+
+	scanner := bufio.NewScanner(strings.NewReader(content))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, domainPrefix) {
+			domain = strings.TrimPrefix(line, domainPrefix)
+		}
+	}
```

Comment:
I'll look into this, I'm not even sure if setting domain is a common thing. Getting options in there would also be nice. I'm debating on looking for a go pkg built for parsing resolv.conf instead of the bespoke functions.